README.md

```diff
@@ -84,36 +84,6 @@ const cc = new Card("1111111111111111",2,18,"123");
 cc.name = "Osei Fortune";
 ```
 
-### Get Token
-
```

Comment:
Removed example as token creation is now deprecated (even if it's still supported)